pkgs/development/compilers/zig/default.nix

```diff
@@ -33,12 +43,12 @@ llvmPackages.stdenv.mkDerivation rec {
   doCheck = true;
 
   meta = with lib; {
+    homepage = "https://ziglang.org/";
     description =
       "General-purpose programming language and toolchain for maintaining robust, optimal, and reusable software";
-    homepage = "https://ziglang.org/";
     license = licenses.mit;
+    maintainers = with maintainers; [ andrewrk AndersonTorres ];
     platforms = platforms.unix;
-    maintainers = [ maintainers.andrewrk ];
     # See https://github.com/NixOS/nixpkgs/issues/86299
     broken = stdenv.isDarwin;
```

Comment:
Can it be done via llvmPackages.stdenv?

Reply:
I don't understand your question. The problem here is that the libc used in nixpkgs on MacOS is too old.

Reply:
@Mic92 

Specifically, `broken = llvmPackages.stdenv.isDarwin;`

Reply:
Should not make a difference actually.